lib/timescaledb/rails/extensions/active_record/schema_dumper.rb

```diff
@@ -19,14 +20,20 @@ def tables(stream)
         def continuous_aggregates(stream)
           return unless timescale_enabled?
 
-          Timescaledb::Rails::ContinuousAggregate.all.each do |continuous_aggregate|
-            continuous_aggregate(continuous_aggregate, stream)
-            continuous_aggregate_policy(continuous_aggregate, stream)
+          deps = Timescaledb::Rails::ContinuousAggregate.find_each.index_by(&:materialization_hypertable_name)
```

Comment:
continuous aggregates can depend on each other, and we need to make sure they're topologically ordered in the dumped schema